lib/main_transactional.pm

```diff
@@ -0,0 +1,283 @@
+    }
+}
+
+sub load_autoyast_installation_tests {
+    loadtest 'autoyast/prepare_profile' if get_var('AUTOYAST_PREPARE_PROFILE');
+    loadtest 'installation/bootloader_start';
+    loadtest 'autoyast/installation';
+    loadtest 'autoyast/console';
+    loadtest 'autoyast/login';
+    loadtest 'autoyast/logs';
+    loadtest 'console/textinfo';
+}
+
+sub load_selfinstall_boot_tests {
+    loadtest 'installation/bootloader_uefi';
+    loadtest 'microos/selfinstall';
+}
+
+sub load_remote_target_tests {
+    loadtest 'installation/bootloader_start';
+    loadtest 'remote/remote_target';
+    loadtest 'console/textinfo';
+    loadtest 'microos/networking';
+    loadtest 'microos/libzypp_config';
+    loadtest 'microos/one_line_checks';
+    loadtest 'microos/services_enabled';
+    loadtest 'console/journal_check';
+    loadtest 'shutdown/shutdown';
+}
+
+sub load_remote_controller_tests {
+    loadtest 'support_server/login';
+    loadtest 'support_server/setup';
+    loadtest 'remote/remote_controller';
+    loadtest 'installation/welcome';
+    if (is_sle_micro) {
+        loadtest 'installation/scc_registration';
+    } elsif (is_microos) {
+        loadtest 'installation/online_repos';
+        loadtest 'installation/installation_mode';
+        loadtest 'installation/system_role';
+    }
+    loadtest 'installation/ntp_config_settings';
+    loadtest 'installation/user_settings_root';
+    loadtest 'installation/resolve_dependency_issues';
+    loadtest 'installation/installation_overview';
+    loadtest 'installation/disable_grub_timeout';
+    loadtest 'installation/start_install';
+    loadtest 'installation/await_install';
+    loadtest 'installation/reboot_after_installation';
+    loadtest 'support_server/wait_children';
+}
+
+sub load_common_tests {
+    loadtest 'console/regproxy' if is_regproxy_required;
+    loadtest 'microos/networking';
+    loadtest 'microos/libzypp_config';
+    loadtest 'microos/image_checks' if is_image;
+    loadtest 'microos/one_line_checks';
+    loadtest 'microos/services_enabled';
+    # MicroOS -old images use wicked, but cockpit-wicked is no longer supported in TW
+    loadtest 'microos/cockpit_service' unless is_staging || (is_microos('Tumbleweed') && get_var('HDD_1') =~ /-old/);
+    # Staging has no access to repos and the MicroOS-DVD does not contain ansible
+    # Ansible test needs Packagehub in SLE and it can't be enabled in SLEM
+    loadtest 'console/ansible' unless (is_staging || is_sle_micro);
+    loadtest 'console/kubeadm' if (check_var('SYSTEM_ROLE', 'kubeadm'));
+}
+
+
+sub load_transactional_tests {
+    loadtest 'transactional/filesystem_ro';
+    loadtest 'transactional/trup_smoke';
+    loadtest 'transactional/transactional_update';
+    loadtest 'transactional/rebootmgr';
+    loadtest 'transactional/health_check';
+}
+
+
+sub load_network_tests {
+    loadtest 'microos/networking';
+    loadtest 'microos/libzypp_config';
+    loadtest 'console/firewalld';
```

Comment:
sle-micro does not come with configured firewall, I think the same applies for ALP. Doesn't it fail? I haven't noticed any enclosed VR with this module

Reply:
You are right, but this section is only executed in ALP tests.

Reply:
Ah, now I see it there. Ok

Reply:
But maybe a comment would be nice here, as I guess there will be changes regarding firewalld in ALP.